src/averageAges.js

```diff
@@ -37,7 +45,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenArray = people.filter(
+    (person, index, arr) => {
```

Comment:
Consider using implicit return and removing unnecessary parameters 'index' and 'arr' from the filter function.